forms/api/service.php

```
@@ -0,0 +1,598 @@
+                $total = (float) $total + (float) $addon['price'];
+                break;
+
+            case AddonType::ALL_SESSIONS_DISCOUNT:
+            case AddonType::EARLY_DISCOUNT:
+                $total = (float) $total - (float) $addon['price'];
+                break;
+        }
+    }
+
+    return (float) $total;
+}
```

PHPMD (2.15.0)

[error] The parameter $selected_addons is not named in camelCase. (undefined)

(CamelCaseParameterName)